app.py

```diff
@@ -120,23 +104,27 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
 db_secret_name = database.pgstac.secret.secret_name
 db_security_group = database.db_security_group
 
+base_api_url = f"https://{veda_app_settings.stage_name()}.{veda_app_settings.veda_custom_host}".strip(
```

Comment:
Just a note that we're passing something similar to this `base_api_url` in both the raster and stac apis via the `VEDA_CUSTOM_HOST` env for parameter mapping: https://github.com/NASA-IMPACT/veda-backend/blob/72f446ae95ae67a57dd41dbc5baca59d02a3cfa0/stac_api/infrastructure/construct.py#L102

`VEDA_CUSTOM_HOST` was always tricky to keep track of because you need to be mindful of the stage name in the url. We could potentially remove this env and pass `base_api_url` to the stac and raster APIs instead.